app/javascript/components/developer/DeleteDevelopers.js

```diff
@@ -0,0 +1,56 @@
+import React, { useEffect, useState } from "react";
+import { useDispatch, useSelector } from "react-redux";
+import { fetchDevelopers } from "../redux/developer";
+import { ToastContainer, toast } from "react-toastify";
+
+import "react-toastify/dist/ReactToastify.css";
+
+function DeleteDevelopers() {
+  const dispatch = useDispatch();
+  const [fetchedDevelopers, setFetchedDevelopers] = useState([]);
+  const developers = useSelector((state) => state.developers.developers);
+
+  useEffect(() => {
+    dispatch(fetchDevelopers());
+    setFetchedDevelopers(developers);
+  }, []);
+  console.log(developers);
```

Comment:
- [ ] Kindly remove this old piece of debugging code in order to improve your app's cleanlines.